config/index.js

```diff
@@ -39,6 +39,7 @@ const systemManagerBackend = new SystemManagerBackend({
   logger
 })
 const backends = {
+  // when adding a new backend, make sure to change the CRD property too
```

Comment:
is this comment still valid, or is it an artifact from a previous revision?

Reply:
its valid :) he had CRD validation in this PR first, but since I had a pr for it as well it was removed from here, but it still applies!